modules/perception/camera/tools/offline/visualizer.cc

```diff
@@ -712,37 +713,67 @@ bool Visualizer::key_handler(const std::string &camera_name, const int key) {
   if (show_help_text_) {
     help_str_ += " (ON)";
     help_str_ += "\nR: reset matrxi\nB: show box";
-    if (show_box_) help_str_ += "(ON)";
+    if (show_box_) {
+      help_str_ += "(ON)";
+    }
     help_str_ += "\nV: show velocity";
-    if (show_velocity_) help_str_ += " (ON)";
+    if (show_velocity_) {
+      help_str_ += " (ON)";
+    }
     help_str_ += "\nC: use class color";
-    if (use_class_color_) help_str_ += " (ON)";
+    if (use_class_color_) {
+      help_str_ += " (ON)";
+    }
     help_str_ += "\nS: capture screen";
     help_str_ += "\nA: capture video";
     help_str_ += "\nI: show type id label";
-    if (show_type_id_label_) help_str_ += " (ON)";
+    if (show_type_id_label_) {
+      help_str_ += " (ON)";
```

Comment:
A lot of temp string objects are created. Maybe `sstream` or `apollo::common::util::StrAppend` is better.

Reply:
Either or,  I prefer to sstream since StrAppend is better for the number of 3 strings, what's your opinion? Thanks!